app/javascript/app/components/emission-pathways/emission-pathways-graph/emission-pathways-graph.js

```diff
@@ -113,8 +105,17 @@ class EmissionPathwayGraphContainer extends PureComponent {
       'indicator',
       'currentLocation'
     ].forEach(f => {
-      if (!search[f] && filtersSelected[f]) {
-        this.updateUrlParam({ name: f, value: filtersSelected[f].value });
+      if (!search[f]) {
+        if (f === 'currentLocation' && filtersSelected.location) {
+          this.updateUrlParam({
```

Comment:
This probably needs be refactored. Calling several times updateURL will only get the one on the URL. Its better if we aggregate them in an array and only call updateUrlParam once as in data-explorer-content.js:
```
data-explorer-content.js:
    const paramsToUpdate = Object.keys(validParams).map(key => ({
      name: key,
      value: validParams[key]
    }));
    this.updateUrlParam(paramsToUpdate, true);
```